canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -35,6 +38,61 @@ $toolbar-divider-width: 1px;
 
 	/* Prevent elements from being dragged onto canvas */
 	-webkit-user-drag: none;
+	&.toolbar-size-small {
+		.toolbar-left-bar {
+			padding: 0;
+			.toolbar-popover-list {
+				top: $spacing-07;
+				right: 0;
+			}
+		}
+		.toolbar-item button {
+			padding: 3px;
+			height: auto;
+		}
+		.toolbar-overflow-item {
+			background-color: $ui-01;
+			display: inline-block;
+			& button {
+				padding-bottom: 11px;
+				padding-left: 3px;
+				height: $toolbar-button-height;
+				background-color: $ui-01;
+				border-right: $toolbar-divider-width solid $ui-03;
+			}
+			& button:hover {
+				background-color: $hover-field;
+			}
+			& button:disabled:hover {
+				background-color: $ui-01;
+			}
+		}
+		.toolbar-overflow-menu-item button {
+			width: 100%;
+			padding-top: 2px;
+			padding-bottom: 2px;
+			min-height: 20px;
```

Comment:
There are many hardcoded `px` values that should not be necessary if you are able to leverage the Carbon styles for buttons. Can you revisit these and see what you can do? Please see this comment https://github.com/elyra-ai/canvas/pull/1143/files#r1009987416

Reply:
I did remove some hardcoded...but I added a few styles to make it identical to the cloud, also you commented about the backgroundColor within `&.toolbar-size-small`, the thing is if i remove it i lose  overflowicon color oh hover 
<img width="382" alt="image" src="https://user-images.githubusercontent.com/89609176/199613320-2c3b0b79-aa64-4ad7-8c5d-0d5ecafe1038.png">

Reply:
All styles need to conform to Carbon standards, you can take a look at what styles are applied for these icons for future reference https://v7-react.carbondesignsystem.com/?path=/story/components-button--playground